components/brave_wallet_ui/utils/buy-asset-url.ts

```diff
@@ -1,21 +1,20 @@
-import { Network, AccountAssetOptionType, UserAccountType } from '../constants/types'
-import { NetworkOptions } from '../options/network-options'
+import { AccountAssetOptionType, UserAccountType } from '../constants/types'
 
 const wyreID = 'AC_MGNVBGHPA9T'
 
-export function BuyAssetUrl (network: Network, asset: AccountAssetOptionType, account: UserAccountType, buyAmount: string) {
-  switch (network) {
-    case Network.Mainnet:
+export function BuyAssetUrl (networkChainId: string, asset: AccountAssetOptionType, account: UserAccountType, buyAmount: string) {
+  switch (networkChainId) {
+    case '0x1':
       return `https://pay.sendwyre.com/?dest=ethereum:${account.address}&destCurrency=${asset.asset.symbol}&amount=${buyAmount}&accountId=${wyreID}&paymentMethod=debit-card`
-    case Network.Ropsten:
+    case '0x3':
       return 'https://faucet.metamask.io/'
-    case Network.Kovan:
+    case '0x2a':
       return 'https://github.com/kovan-testnet/faucet'
-    case Network.Rinkeby:
+    case '0x4':
       return 'https://www.rinkeby.io/'
-    case Network.Goerli:
+    case '0x5':
       return 'https://goerli-faucet.slock.it/'
     default:
-      throw new Error(`Unknown cryptocurrency exchange or faucet: "${NetworkOptions[network].name}"`)
+      return ''
```

Comment:
@Douglashdaniel as we discussed I return empty here for custom networks to hide UI in future PRs